packages/model/addon/-private/errors.js

```diff
@@ -391,7 +402,8 @@ export default ArrayProxy.extend(DeprecatedEvented, {
    errors.get('messages')
    // => []
    ```
-   @method remove
+   @method clear
+    @public
```

Comment:
can't tell if this is a UI issue or there is an actual extra space here

Reply:
just a copy pasta issue